packages/url_launcher/url_launcher/README.md

```diff
@@ -49,6 +49,42 @@ void _launchURL() async =>
     await canLaunch(_url) ? await launch(_url) : throw 'Could not launch $_url';
 ```
 
+### Android
+
+Starting from API 30 Android requires package visibility configuration in your `AndroidManifest.xml` otherwise `canLaunch` would return `false`. Please see the `<queries>` element example below and tailor to your needs. For more information see the [Managing package visibility documentation](https://developer.android.com/training/basics/intents/package-visibility).
```

Comment:
Nit: The capitalization of "managing" seems random; that's not even the name of the page. How about just "the [package visibility documentation]"

Reply:
Actually I took the initiative to use the exact title of that page "Package visibility filtering on Android". Let me know and I can modify it.